scada/scada.go

```diff
@@ -0,0 +1,238 @@
+package scada
+
+import (
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"sync"
+	"time"
+
+	sc "github.com/hashicorp/scada-client"
+)
+
+const (
+	InfrastructureResource = "infrastructures"
+	BoxesResource          = "boxes"
+	SharesResource         = "shares"
+	NomadClusterResource   = "nomad-cluster"
```

Comment:
These can be left out - we shouldn't need to update the client library to add new resource types later on.